docs/development/test-expansion-plan.md

```
@@ -0,0 +1,630 @@
+artifacts produced by a different build of gcc/clang — surfaces assumptions about
+compiler-specific DWARF layout.
+
+**Spec:**
+
+1. **Curate binary pairs.** Clone libabigail source (tag `libabigail-2.6` or
+   latest stable). Identify subdirectories under `tests/data/` containing paired
+   `.so` files with corresponding expected output (XML or text diff).
+
+2. **Create fixture directory.** Store curated pairs under
+   `tests/fixtures/libabigail/` with the following layout:
+   ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)